docs/releases.md

```diff
@@ -6,10 +6,122 @@ sidebar_label: Releases
 
 ------
 
-## 2.7.0 - Mar 16 2021
+## 2.9.0 - May 15 2021
 
 <br><font size="4">Latest Release</font><br/> (Recommended)<br/>
 
+OpenEBS v2.9 is another maintenance release before moving towards 3.0 primarily focusing on enhancing the E2e tests and build/release workflows. This release includes fixes for user-reported critical bugs as well as fixes and enhancements to improve the E2e test coverage. There has been some significant progress made on the alpha features as well. 
```

Comment:
Why E2e is like this way? better to go with E2E?

Reply:
all are valid. But we can probably take this uniformity across all in one PR.